strawberryfields/apps/qchem/dynamics.py

```diff
@@ -117,3 +122,92 @@ def op(t, Ul, w, q):
         sf.ops.Interferometer(Ul) | q
 
     return op
+
+
+def sample_fock(
+    input_state: list,
+    t: float,
+    Ul: np.ndarray,
+    w: np.ndarray,
+    n_samples: int = 1,
+    loss: float = 0.0,
+    cutoff: int = 15,
```

Comment:
It's probably not a good idea to have a default cutoff. I think Tom mentioned this before? It's important enough that the user should probably give it some thought instead of relying on default values